scripts/build/download-tool.sh

```diff
@@ -78,7 +78,8 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+echo "Downloading $ARGV_URL"
+curl --continue-at - --retry 100 --location --output "$TEMP_OUTPUT" "$ARGV_URL"
```

Comment:
Fair enough \*shrug\*. But in that case, I think that everywhere we use `curl` we ought to call it with the same options. But I wonder if a retry of 100 is a little excessive?